parsec/core/gui/claim_device_widget.py

```diff
@@ -380,6 +381,9 @@ def __init__(self, jobs_ctx, claimer, device_email):
         self.label_wait.hide()
         self.button_ok.clicked.connect(self._on_claim_clicked)
 
+    def _on_device_text_changed(self, device_text):
+        self.widget_password.user_inputs = [device_text]
```

Comment:
I don't understand why we want to use the device name as password...

 edit: I understand now (we use the non-password inputs to check the security of the password), but it's not really clear at first look
We should add a comment here about this